### PR TITLE
Add Testing Instructions for Inbox 2.0

### PR DESCRIPTION
**Pull request by adrianduffell**
Adds testing instructions for Inbox 2.0.

No changelog.

### CODE REVIEW

File: TESTING-INSTRUCTIONS.md [hunk 1]
```diff
@@ -12,6 +12,48 @@
...
+### Inbox - Activity menu
+
+1. Go to WooCommerce products screen
+2. On Menu bar, see that "Activity" menu item exists in top right corner.
+3. Click "Activity" menu item.
+4. See notes appear in panel.
+
+### Inbox - Dismiss all notes
+
+1. Go to WooCommerce products screen
```
Comment (ilyasfoo) [inline TESTING-INSTRUCTIONS.md:49]: Did you mean homescreen here?

**Review by ilyasfoo — COMMENTED** [review]
I think the instructions covers it pretty well! Just have a couple of comments

File: TESTING-INSTRUCTIONS.md (hunk 1 above)
Comment (adrianduffell) [reply TESTING-INSTRUCTIONS.md]: Yes, thanks! Fixed in 3e55eda.

File: TESTING-INSTRUCTIONS.md [hunk 2]
```diff
@@ -12,6 +12,48 @@
 
 ## 3.0.0
 
+### Inbox - 320 character limit
+
+On a new site, with English language settings:
+
+1. Go to WooCommerce home screen
+2. See that all Inbox notes are short in length (aproximately less than 320 characters).
```
[the remark this answers is not in the document]
Comment (adrianduffell) [reply TESTING-INSTRUCTIONS.md:20]: Ah of course, good catch. Perhaps I should add it to 3.1.0 testing instructions? 🤔

[the remark this answers is not in the document]
Comment (adrianduffell) [reply TESTING-INSTRUCTIONS.md:20]: Moved the instructions to 3.1.0 in 4c2ed7f

**Review by ilyasfoo — APPROVED** [review]
Nice work! LGTM! 🚢

**Pull request merged**